docs/snippets/getting-started/anta_nrfu_text.sh

```diff
@@ -0,0 +1,9 @@
+anta nrfu \
+    --username arista \
+    --password $LABPASSPHRASE \
+    --inventory ./inventory.yml \
+    `# uncomment the two next lines if you have an enable password` \
+    `# --enable` \
+    `# --enable-password <password>` \
```

Comment:
Do we need the backticks here?

Reply:
what do you think? :p

Reply:
basically it allows to execute the command without having to move the comments out (#ninja trick)
this way we can run the shell script ourselves :)